config/initializers/omniauth.rb

```diff
@@ -36,7 +38,9 @@
 
   # SAML strategy
   if ENV['SAML_ENABLED'] == 'true'
-    saml_options = options
+    saml_options = {}
+    saml_options[:redirect_at_sign_in] = ENV['OAUTH_REDIRECT_AT_SIGN_IN'] == 'true'
```

Comment:
Drop that line, `ENV['OAUTH_REDIRECT_AT_SIGN_IN']` / `cas_options[:redirect_at_sign_in]` has been removed, in favor of `ENV['OMNIAUTH_ONLY']` which is handled independently